### ISSUE TITLE
Fix version ordering on network monitor

### ISSUE THREAD

**Issue by diego-G** [LiskHQ/lisk-explorer#449]
### Expected behaviour

Selecting version desc ordering should have the next result:
```
0.9.12a
0.9.11a
0.9.10b
0.9.10a
0.9.9a
```

### Actual behaviour
```
0.9.9a
0.9.12a
0.9.11a
0.9.10b
```

### Steps to reproduce

Ordering by version https://testnet-explorer.lisk.io/networkMonitor

**Comment by MichalTuleja** [comment]
It looks like the problem can be solved only on the frontend side. I can provide separate integer property on the backend side to ensure sorting order, but the frontend implementation should be changed anyway in order to use that property. I think the best practice would be to implement a custom sorting function using a controller or even a view abstraction level.

**Comment by reyraa** [comment]
This issue has been fixed and merged in #336. I'll have a look to see what has changed since then.
